packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/share-my-ads-socials/share-my-ads-socials.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { FacebookShareButton, TelegramShareButton, TwitterShareButton, WhatsappShareButton } from 'react-share';
+import { Icon, Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+
+const ShareMyAdsIcons = () => {
+    // TODO: replace with proper url when available
+    const advert_url = window.location.href;
+
+    return (
+        <div className='share-my-ads-icons'>
+            <WhatsappShareButton url={advert_url}>
+                <div className='share-my-ads-icons__circle'>
+                    <Icon className='share-my-ads-icons__circle-icon' icon='IcWhatsappFilled' size={34} />
+                </div>
+                <Text size='xxxs'>
+                    <Localize i18n_default_text='WhatsApp' />
+                </Text>
+            </WhatsappShareButton>
+            <FacebookShareButton url={advert_url}>
+                <div className='share-my-ads-icons__circle'>
+                    <Icon className='share-my-ads-icons__circle-icon' icon='IcFacebook' size={34} />
+                </div>
+                <Text size='xxxs'>
+                    <Localize i18n_default_text='Facebook' />
+                </Text>
+            </FacebookShareButton>
+            <TelegramShareButton url={advert_url}>
+                <div className='share-my-ads-icons__circle'>
+                    <Icon className='share-my-ads-icons__circle-icon' icon='IcTelegram' size={34} />
+                </div>
+                <Text size='xxxs'>
+                    <Localize i18n_default_text='Telegram' />
+                </Text>
+            </TelegramShareButton>
```

Comment:
Just a thought, can we do something like this instead. Because many properties are being repeated so many times 🤔 

```suggestion
           const shareButtons = [
  { ShareButton: WhatsappShareButton, icon: 'IcWhatsappFilled', text: 'WhatsApp' },
  { ShareButton: FacebookShareButton, icon: 'IcFacebook', text: 'Facebook' },
  { ShareButton: TelegramShareButton, icon: 'IcTelegram', text: 'Telegram' },
];

<div className='share-my-ads-icons'>
  {shareButtons.map(({ ShareButton, icon, text }) => (
    <ShareButton key={text} url={advert_url}>
      <div className='share-my-ads-icons__circle'>
        <Icon className='share-my-ads-icons__circle-icon' icon={icon} size={34} />
      </div>
      <Text size='xxxs'>
        <Localize i18n_default_text={text} />
      </Text>
    </ShareButton>
  ))}
</div>;

```

Reply:
sure man, i can see if this would work

Reply:
Still unsure whether Localize will work in this case or not. Lemme know 🙏

Reply:
i think it would be fine regarding the Localize component, since we don't need to translate the app name right? 👀

Reply:
Oh yeah youre right, onz then